### PR TITLE
Feature 625 additional custom event tracking

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/op92KSWU/625-additional-custom-event-tracking-using-google-analytics

### CODE REVIEW

File: assets/js/components/contacts.js [hunk 1]
```diff
@@ -109,6 +118,21 @@ export class ContactSection {
...
+        category: "Contacts",
+        action: "Click",
+        label: `${$(e.target).text()}`,
+      });
+    }));
+    this.$muniLinks.on('click', ((e) => {
+      gtag('event', 'contact_info', {
+        category: "Contacts",
+        action: "Click",
+        label: `Municipal wide contacts`,
```
Comment (paulmwatson) [inline assets/js/components/contacts.js:134]: Don't need `` if no variable interpolation.

File: assets/js/components/contacts.js [hunk 2]
```diff
@@ -109,6 +118,21 @@ export class ContactSection {
...
+      gtag('event', 'contact_info', {
+        category: "Contacts",
+        action: "Click",
+        label: `${$(e.target).text()}`,
+      });
+    }));
+    this.$muniLinks.on('click', ((e) => {
+      gtag('event', 'contact_info', {
+        category: "Contacts",
+        action: "Click",
```
Comment (paulmwatson) [inline assets/js/components/contacts.js:133]: A bit repetitive, just the target and label different. Not major though.

File: assets/js/components/contacts.js (hunk 1 above)
Comment (michaelglenister) [reply assets/js/components/contacts.js]: Changed literal to regular string

File: assets/js/components/contacts.js (hunk 2 above)
Comment (michaelglenister) [reply assets/js/components/contacts.js]: Moved these tags to a function

**Review by paulmwatson — APPROVED** [review]

**Pull request merged**